tests/components/alexa/test_capabilities.py

```diff
@@ -243,6 +243,97 @@ async def test_api_select_input(
     assert call.data["source"] == source_list[idx]
 
 
+@pytest.mark.parametrize(
+    (
+        "target_activity",
+        "activity_list",
+        "current_activity_index",
+        "target_activity_index",
+    ),
+    [
+        ("TV", ["TV", "MUSIC", "DVD"], 1, 0),
+        ("MUSIC", ["TV", "MUSIC", "DVD", 1000], 0, 1),
+        ("DVD", ["TV", "MUSIC", "DVD", None], 0, 2),
+        ("BAD DEVICE", ["TV", "MUSIC", "DVD"], 0, None),
+    ],
+)
```

Comment:
Please add cases with zero and one activity

Reply:
Done.